src/core.js

```diff
@@ -884,11 +884,10 @@ $.extend( $.validator, {
 
 					// If the error is a label, then associate using 'for'
 					error.attr( "for", elementID );
-				}
 
-				// If the element is not a child of an associated label, then it's necessary
-				// to explicitly apply aria-describedby
-				else if ( error.parents( "label[for='" + this.escapeCssMeta( elementID ) + "']" ).length === 0 ) {
+					// If the element is not a child of an associated label, then it's necessary
+					// to explicitly apply aria-describedby
+				} else if ( error.parents( "label[for='" + this.escapeCssMeta( elementID ) + "']" ).length === 0 ) {
```

Comment:
This fix a CS issue reported in https://github.com/jzaefferer/jquery-validation/issues/1618#issuecomment-159093885